source/common/http/conn_manager_impl.h

```diff
@@ -173,8 +173,8 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
     // either because [de|en]codeHeaders() of the current filter returns StopAllIteration or because
     // [de|en]codeHeaders() adds new metadata to [de|en]code, but we don't know
     // [de|en]codeHeaders()'s return value yet. The storage is created on demand.
-    std::unique_ptr<MetadataMapVector> saved_request_metadata_;
-    std::unique_ptr<MetadataMapVector> saved_response_metadata_;
+    std::unique_ptr<MetadataMapVector> saved_request_metadata_ = nullptr;
+    std::unique_ptr<MetadataMapVector> saved_response_metadata_ = nullptr;
```

Comment:
Is this necessary?  If so let's use the {nullptr} syntax you use for request_medata_map_vector_

Reply:
Fixed. Thanks!